Terminal.Gui/Drivers/V2/ApplicationV2.cs

```diff
@@ -92,13 +93,17 @@ private void CreateDriver (string? driverName)
         {
             _coordinator = CreateNetSubcomponents ();
         }
+        else if (definetlyUnix)
+        {
+            _coordinator = CreateUnixSubcomponents ();
+        }
         else if (p == PlatformID.Win32NT || p == PlatformID.Win32S || p == PlatformID.Win32Windows)
         {
             _coordinator = CreateWindowsSubcomponents ();
         }
         else
         {
-            _coordinator = CreateNetSubcomponents ();
+            _coordinator = CreateUnixSubcomponents ();
```

Comment:
This changes the default driver on linux/macos to the new unix one. Worth noting.